CONTRIBUTING.md

```diff
@@ -0,0 +1,91 @@
+# Contributing to `ptvsd` 
+
+[![Build Status](https://ptvsd.visualstudio.com/_apis/public/build/definitions/557bd35a-f98d-4c49-9bc9-c7d548f78e4d/1/badge)](https://ptvsd.visualstudio.com/ptvsd/ptvsd%20Team/_build/index?definitionId=1)
+[![Build Status](https://travis-ci.org/Microsoft/ptvsd.svg?branch=master)](https://travis-ci.org/Microsoft/ptvsd)
+[![GitHub](https://img.shields.io/badge/license-MIT-brightgreen.svg)](https://raw.githubusercontent.com/Microsoft/ptvsd/master/LICENSE)
+[![PyPI](https://img.shields.io/pypi/v/ptvsd.svg)](https://pypi.org/project/ptvsd/)
+
+
+## Contributing a pull request
+This project welcomes contributions and suggestions.  Most contributions require you to agree to a
+Contributor License Agreement (CLA) declaring that you have the right to, and actually do, grant us
+the rights to use your contribution. For details, visit https://cla.microsoft.com.
+
+When you submit a pull request, a CLA-bot will automatically determine whether you need to provide
+a CLA and decorate the PR appropriately (e.g., label, comment). Simply follow the instructions
+provided by the bot. You will only need to do this once across all repos using our CLA.
+
+### Prerequisites
+These packages are needed to to run `ptvsd` tests:
+* setuptools
+* coverage
+* requests
+* flask
+* django
+* pytest
```

Comment:
Also `pytest-timeout`.

For pytests, I made a requirements.txt that can be used with `pip install -r`. Maybe make one on the top level - e.g. test_requirements.txt - and put the combined list there?

Reply:
I will add a `test_requirements.txt`.